src/funnels/automation/VaultMinter.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
We also need to add a paragraph in the README (can do next to the ConduitMover one.)

Reply:
Done. Let me know if you have something better to put here.

Reply:
looks good.